blueprints/ember-frost-core/index.js

```diff
@@ -1,36 +1,4 @@
-const blueprintHelper = require('ember-frost-core/blueprint-helper')
-
```

Comment:
We can probably delete the blueprint-helper and related modules.

Reply:
We will but we cannot yet until we fix the dependencies/blueprints in the other frost addons since they are using the blueprintHelper. The plan is once we have the other addons fixed we will circle back and remove it.

Reply:
#521